index.html

```diff
@@ -62,5 +62,198 @@ <h1>Hi👋, I'm Cephas Tokoli</h1>
+                            <p>JavaScript</p>
+                        </div>
+                        <div class="details-button">
+                        <div>
+                            <p >See this project</p>
+                            <img src="images/Union.svg" alt="union">
+                        </div>
+                        </div>
+                        </div>
+                    </div>
+                </div>
+        
+                <div class="project">
+                    <div class="laptop-img">
+                        <img src="images/laptop5.png" alt="laptop">
+                    </div>
+                    <div class="project-details">
+                        <div class="tech-details">
+                            <h2 class="project-name">Project name goes here</h2>
+                        <div class="tech-stack">
+                            <p>HTML/CSS</p>
+                            <p>Ruby on Rails</p>
+                            <p>JavaScript</p>
+                        </div>
+                        <div class="details-button">
+                        <div>
+                            <p >See this project</p>
+                            <img src="images/Union.svg" alt="union">
+                        </div>
+                        </div>
+                        </div>
+                    </div>
+                </div>
+        </section>
+        <section class="about">
+            <div>
+                <h2 class="headings">About</h2>
+                <div class="content">
+                <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+                <button class="button download">Get my CV
+                    <div class="icon"></div>
+                </button>
+                </div>
+            </div>
+        </section>
+        <section >
+            <div class="competency">
+                <div class="language grid-item">
+                    <div class="grid-item-head">
+                        <ul>
+                            <li>Languages</li>
+                        </ul>
+                    </div>
+                    <ul>
+                        <li>Javascript(ES5/ES6)</li>
+                        <li>Ruby</li>
+                        <li>CSS</li>
+                        <li>HTML</li>
+                    </ul>
+                </div>
+                <div class="frameworks grid-item">
+                   <div class="grid-item-head">
+                    <ul>
+                        <li>Frameworks</li>
+                    </ul>
+                   </div>
+                    <ul>
+                        <li>React.js</li>
+                        <li>Ruby on rails</li>
+                        <li>RSpec</li>
+                        <li>Capibara</li>
+                    </ul>
+                </div>
```

Comment:
As requested by the previous reviewer, please let's use `ul` and `li` for categories in the "About" section. `Languages`, `Frameworks`, and `Skills` should be list items.

i.e

```HTML
<ul>

  <li>
    <h3>Languages</h3>
    <ul>
      <li>Javascript(ES5/ES6)</li>
      <li>Ruby</li>
      <li>CSS</li>
      <li>HTML</li>
    </ul>
  </li>

  <li>
    <h3>Frameworks</h3>
    <ul>
      <li>. . . </li>
      <li>. . . </li>
    </ul>
  </li>

  <li>
    <h3>Skills</h3>
    <ul>
      <li>. . . </li>
      <li>. . . </li>
    </ul>
  </li>

</ul>
```